### DISCUSSION TITLE
页眉中的中文标题带有英文字符（默认为大写字符），请问有什么办法将英文字符改为小写字符吗？

### DISCUSSION THREAD

**Discussion by SMX-xin** [sjtug/SJTUThesis#603] [提问] [answered]
**问题描述**:
按照标准的论文格式来说，当页眉中中文标题带有英文字符时，英文字符规范是大写，但是我毕业设计老师要求改为小写，自己网页上找了一些解决方法都不太能行，自己也是latex初学者，不太明白怎么搞，请问有解决办法？

**系统信息**:
windows  latex在线版overleaf
- OS版本(带版本号): macOS/Windows/Linux
- TeX 发行版(带版本号): TeX Live/MiKTeX/MacTeX

**日志信息**:

**Comment by Yu3H0** [comment, marked as answer]
使用\lowercase{}，将要小写的英文字符放在括号里即可。
